pkg/common/health/health.go

```diff
@@ -20,6 +20,10 @@ const (
 
 // State is the health state of a subsystem.
 type State struct {
+	// Started is whether the subsystem is finished starting.
+	// if undefined, it is treated as started=true.
```

Comment:
Could you elaborate on why the undefined state is needed?

Reply:
For backwards compatibility with existing code. For example,  the spire-server code in this pr still doesn't set whether it is started or not, so it implies started=true so the spire-server behaves as it always has.

Wasn't sure all the places it may be used. We could update them all if that was preferable.